autoscaler/kube.py

```diff
@@ -33,6 +33,9 @@ def __init__(self, pod):
         self.status = pod.obj['status']['phase']
         self.uid = metadata['uid']
         self.selectors = pod.obj['spec'].get('nodeSelector', {})
+        # TODO: Remove this, after everyone has migrated off reservations
```

Comment:
do you want to update the underlying pod here? otherwise, it still won't get scheduled?